### PR TITLE
[65] Create &Style the Customer Signup Page using CSS- Frontend

### PR DESCRIPTION
**Pull request by gracioust**
## Task: `Create &Style the Customer Signup Page using CSS- Frontend`
- `Create &Style the Customer Signup Page using CSS- Frontend`
signup 1
signup 2
signup 3
signup 4

### CODE REVIEW

**Review by adaugochi — CHANGES_REQUESTED** [review]
Kindly give html elements `class` or `id` names and style them instead of styling their `tag` name. except for cases like `a` tag where you don't want all `a` tag to have `text-decoration`

File: submissions/tosinIshola/HTML/Ecommerce.html [hunk 1]
```diff
@@ -7,15 +7,20 @@
 <body>
 
 <header>
+    <img src="../HTML/Images/fashion.jpg" alt="logo" style="border: 1px solid #ddd;
```
Comment (ToluAkin) [inline submissions/tosinIshola/HTML/Ecommerce.html:10]: @gracioust Move the style here to the external css file

File: submissions/tosinIshola/html-forms/contact_form.html [hunk 2]
```diff
@@ -5,13 +5,20 @@
     <title>CONTACT FORM</title>
 </head>
 <body>
-
+<header>
+    <img src="../HTML/Images/fashion.jpg" alt="logo" style="border: 1px solid #ddd;
```
Comment (ToluAkin) [inline submissions/tosinIshola/html-forms/contact_form.html:9]: @gracioust Move the style here to the external css file

File: submissions/tosinIshola/html-forms/signup.html [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>SIGN UP</title>
+    <link href="../CSS/signup.css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+</head>
+<body>
+<header>
+    <!-- website logo image -->
+    <div id="logo">
+    <img src="../HTML/Images/fashion.jpg" alt="logo" style="border: 1px solid #ddd;
```
Comment (ToluAkin) [inline submissions/tosinIshola/html-forms/signup.html:14]: @gracioust Move the style here to the external css file

**Review by ToluAkin — CHANGES_REQUESTED** [review]

**Pull request closed without merging**